lib/fluent/formatter.rb

```diff
@@ -131,8 +131,22 @@ class JSONFormatter < Formatter
       include HandleTagAndTimeMixin
       include StructuredFormatMixin
 
+      config_param :disable_oj, :bool, :default => false
```

Comment:
I don't want to support more parsers and I want to remove this option in the feature.
So I don't want to use more better name because users may misunderstands fluentd will support another parsers.